packages/ui/components/ui/select.tsx

```diff
@@ -26,10 +26,7 @@ const SelectTrigger = React.forwardRef<
     {children}
     <SelectPrimitive.Icon asChild>
       <ChevronDownIcon
-        className={cn(
-          'h-6 w-6 text-muted transition-all duration-500 ease-bounce',
```

Comment:
ESLint was complaining that `ease-bounce` isn't a valid class — should it be [`animate-bounce`](https://github.com/jamiebuilds/tailwindcss-animate?tab=readme-ov-file#basic-usage)?